modules/aws_asg/high-cpu-check.bash

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+__logger() {
+  level="${1}" && shift
+  msg="${1}" && shift
+  date="$(date --iso-8601=seconds)"
+  log_msg="tag=cron time=${date} level=${level} msg=\"${msg}\""
+  for bit in "${@}"; do
+    log_msg="${log_msg} ${bit}"
+  done
+
+  logger -t "$(basename "${0}")" "${log_msg}"
+}
+
+__die() {
+  local status="${1}"
+  local code="${2}"
+  __logger "info" \
+    "cron finished" \
+    "status=${status}"
+  exit "${code}"
+}
+
+report_greedy_containers() {
+  max_cpu="${1}"
+  : "${RUNDIR:=/var/tmp/travis-run.d}"
+  instance_id="$(cat "${RUNDIR}/instance-id")"
+  instance_ip="$(cat "${RUNDIR}/instance-ipv4")"
```

Comment:
I'm sorry for not mentioning this earlier, but I think `max_cpu`, `instance_id`, and `instance_ip` should all be declared as `local`.